### DISCUSSION TITLE
Creative world rules

### DISCUSSION THREAD

**Discussion by wmattiso** [GriefPrevention/GriefPrevention#1430] [Q&A] [answered]
Hello, I was trying to do something on my server where an area in my world can't be built on until it has been claimed, but I want this for survival mode, and the creative world settings restrict things which are crucial for survival, but are fine in creative, such as blocks dropping and dropping items and such. Is there a way that I can modify the rules for a specific creative world, or another option that would be similar?
Thanks!

**Comment by Jikoo** [comment, marked as answer]
See [world configuration](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on) - you probably want `SurvivalRequiringClaims` mode instead.

**Reply by wmattiso** [reply]
This is exactly what I was looking for, thank you so much!
